src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -66,6 +66,10 @@
       TODO: add OptimizeAssemblies="lib/netstandard2.0/System.Threading.Tasks.Extensions.dll"
     -->
     <ExpectedDependency Include="System.Threading.Tasks.Extensions" />
+
+    <ExpectedDependency Include="Microsoft.Win32.Registry"/>
```

Comment:
We probably needs somebody to confirm this change is OK.

Reply:
@tmat could you please take a look if it is a valid update of `ExpectedDependency`?